content/en/monitors/monitor_types/outlier.md

```diff
@@ -16,133 +16,89 @@ further_reading:
-
-{{< img src="monitors/monitor_types/outliers/outliers-algorithm-selector.png" alt="outliers algorithm selector" responsive="true" style="width:80%;">}}
+Outlier Detection is an algorithmic feature that allows you to detect when some members of a group are behaving different compared to the others. For example, you could detect that one web server in a pool is processing an unusual number of requests, or significantly more 500 errors are happening in one AWS availability zone than the others.
 
-This applies the outliers function to your graph, and you'll see any outliers in the group highlighted in bold, warm colors.
-
-{{< img src="monitors/monitor_types/outliers/outliers-algorithm-annotated-newer.png" alt="outliers algorithm annotated newer" responsive="true" style="width:80%;">}}
+{{< img src="monitors/monitor_types/outliers/outliers-metric-alert.png" alt="outliers metric alert" responsive="true" style="width:80%;">}}
 
-There are several outlier detection algorithms you can choose. The default algorithm (DBSCAN) and parameter values should work for most scenarios. However, if you see too many or too few outliers identified, you can tune the algorithm or try an alternate algorithm. To learn more, see the "Outlier Algorithms and Parameters" section below.
+## Monitor creation
 
-### Alert on Outliers
+To create an [outlier monitor][1] in Datadog, use the main navigation: *Monitors --> New Monitor --> Outlier*.
 
-You can also define a monitor to alert when an outlier is detected in an important group.
+The outlier monitor uses the `outliers` function from the Datadog query language. When you apply this function to a series, it returns the usual results with an outlier series marked.
 
-{{< img src="monitors/monitor_types/outliers/outliers-alert-snapshot.png" alt="outliers alert snapshot" responsive="true" style="width:80%;">}}
+### Define the metric
 
-For example, to alert when a Cassandra host is abnormally loaded compared to the rest of the group, you can [add a new outlier monitor][1] for the metric.
+Any metric currently reporting to Datadog is available for monitors. For more information, see the [Metric Monitor][2] page.
 
-Navigate to the [New Monitor][2] page and click **Outlier**. Then fill out the **Define the metric** section just as you would for any other monitor.
+The outlier monitor requires a metric with a group (hosts, availability zones, partitions, etc.) that has three or more members, which exhibit uniform behavior.
 
-{{< img src="monitors/monitor_types/outliers/outliers-new-monitor-define-metric.png" alt="outliers new monitor define metric" responsive="true" style="width:80%;">}}
+### Set alert conditions
 
-In the [alert conditions][3], select the grouping and timeframe. Then select an algorithm and parameter values to use for outlier detection.
+* Trigger a separate alert for each outlier `<GROUP>`
```

Comment:
```suggestion
* Trigger a separate alert for each outlier `<TAG_VALUE>`
```

Reply:
I'm ok with changing this to `<TAG_KEY>` but `<TAG_VALUE>` is incorrect.

Reply:
It might be better to just say `<TAG>` there no that i'm re-reading this.

Reply:
I'm going to leave this as group since it says `(select group)` in the UI.